components/script/dom/bindings/htmlconstructor.rs

```diff
@@ -225,10 +225,18 @@ unsafe fn html_constructor(
         None => {
             // Step 8.1
             let name = QualName::new(None, ns!(html), definition.local_name.clone());
+            // Any prototype used to create these elements will be overwritten before returning
```

Comment:
Now that we have `get_desired_proto` could we replace step 6 above (and the workaround) with a call to that. A related change was made in Gecko here: https://phabricator.services.mozilla.com/D28518

Reply:
I'll file a follow-up issue for this.